branchpro/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""branchpro is a Branching Processes modelling library.
+It contains functionality for modelling, simulating, and visualising the
+number of cases of infections by day during an outbreak of the influenza virus.
```

Comment:
Maybe we will need a copyright or license mention here as well? @DavAug , is this something you thought about?

Also, on their init file Pints has `# Root of the pints module. # Provides access to all shared functionality (optimisation, mcmc, etc.).` Maybe we should add another comment similar to this.

Reply:
Just added with the last commit!